go/keybase/main.go

```diff
@@ -123,7 +123,12 @@ func mainInner(g *libkb.GlobalContext) error {
 		return err
 	}
 
-	return cmd.Run()
+	err = cmd.Run()
+	if !cl.IsService() {
+		// Errors that come up in printing this warning are logged but ignored.
+		printOutOfDateWarnings(g)
+	}
+	return err
```

Comment:
This approach has us making an RPC call during client shutdown. I'm not actually sure it's safe. Would like to think it over with you, @maxtaco.